softserve/views.py

```diff
@@ -43,16 +44,23 @@ def authorized(access_token):
         user_data = github.get('user')
         user = User.query.filter_by(username=user_data['login']).first()
 
+        # retreive key from github account
+        key = requests.get('https://github.com/%s.keys'
```

Comment:
You should perhaps handle a failure here. i.e. if you get something other than a 200 OK.